wp-content/themes/reconasia/assets/_scss/components/_post-block-related.scss

```diff
@@ -0,0 +1,32 @@
+@use 'abstracts' as *;
+
+.post-block-related {
+  display: block;
+  margin-top: rem(32);
```

Comment:
I would remove the margin-top here and let your grid handle the spacing between the post blocks.